### PR TITLE
Exercise 2

### PR DESCRIPTION
**Pull request by FuZzledd**
Please make sure you are raising this PR against your own repository and not the original. If it says https://github.com/CorndelWithSoftwire in the search bar right now, you are in the wrong place!

Please also don't forget to submit the PR's url to Aptem afterwards (by clicking on the corresponding exercise component, pasting the url in the textbox that appears and then pressing the finish button).

### CODE REVIEW

File: todo_app/data/trello_items.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
+from flask import session
```
Comment (CharlieCumber) [inline todo_app/data/trello_items.py]: This import is no longer used.

File: todo_app/data/trello_items.py [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    """
+    Fetches the saved item with the specified ID.
+
+    Args:
+        id: The ID of the item.
+
+    Returns:
+        item: The saved item, or None if no items match the specified ID.
+    """
+    items = get_items()
```
Comment (CharlieCumber) [inline todo_app/data/trello_items.py]: `get_items()` is not defined, nor is the `items` variable used - this whole line can be deleted.

File: todo_app/data/trello_items.py [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    Adds a new item with the specified title to the session.
+
+    Args:
+        title: The title of the item.
+        list_id: The id of the list to add the item to.
+
+    Returns:
+        item: The saved item.
+    """
+    board_id = os.getenv("TRELLO_BOARD_ID")
```
Comment (CharlieCumber) [inline todo_app/data/trello_items.py]: This variable is not used within the scope of this function.

File: todo_app/data/trello_items.py [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+        id: The id of the item.
+    """
+    api_key = os.getenv("TRELLO_API_KEY")
+    api_token = os.getenv("TRELLO_API_TOKEN")
+
+    url = f"https://api.trello.com/1/cards/{id}"
+
+    query = {"key": api_key, "token": api_token}
+
+    response = requests.request("DELETE", url, params=query)
```
Comment (CharlieCumber) [inline todo_app/data/trello_items.py]: The `response` variable has a value set but is not used.

**Review by CharlieCumber — CHANGES_REQUESTED** [review]
I have managed to get your code running locally - and all looks functional.

Please can you add instructions to the README regarding the Trello environment variables such that anyone can set up your code locally?

File: todo_app/app.py [hunk 5]
```diff
@@ -1,11 +1,53 @@
...
+from flask import Flask, render_template, redirect, url_for, request
+
+from todo_app.data import trello_items
 
 from todo_app.flask_config import Config
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
+class Item:
```
Comment (CorndelDevOps) [inline todo_app/app.py]: Usually, classes go in separate files

**Review by CorndelDevOps — APPROVED** [review]
Looks good to me, just one small suggestion

**Pull request merged**